.github/workflows/stale.yml

```diff
@@ -4,8 +4,14 @@ on:
   schedule:
     - cron: "0 * * * *"
 
+permissions:
```

Comment:
This should be set in repo settings, not here.

Reply:
Took care of it. Thanks